client-gateway/bot/passenger.py

```
@@ -370,67 +221,243 @@ def handle_order_status(call):
+        
+        if result.get('success'):
+            trip_data = result.get('trip', {})
+            status = trip_data.get('status', 'UNKNOWN').upper()
+            pickup = trip_data.get('pickup', 'Не вказано')
+            dropoff = trip_data.get('dropoff', 'Не вказано')
+            created_at = trip_data.get('created_at', '')
+            driver_id = trip_data.get('driver_id')
+            
+            status_mapping = {
+                'PENDING': ('⏳ Очікує водія', 'Ваше замовлення в черзі. Очікуємо підтвердження від водія.'),
+                'CONFIRMED': ('✅ Підтверджено', 'Водій прийняв замовлення і прямує до вас.'),
```

Ruff (0.14.10)

String contains ambiguous `і` (CYRILLIC SMALL LETTER BYELORUSSIAN-UKRAINIAN I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)